volcanoes/Volcano from Space.ipynb

```diff
@@ -939,7 +939,7 @@
     "\n",
     "In this exercise we observed the eruption of the biggest European volcano, Etna. We used and visually analyzed images acquired from Sentinel-2 and Landsat 8 satellites. True color image was difficult to interpret thus we combined images acquired in other parts of electromagnetic spectrum into composite image. This enabled us to distinguish lava from soil and snow from smoke and clouds. Combining images from two different satellites gave us more information about the dynamic of the process but interpretation has to include the differences between the sensors.\n",
     "\n",
-    "We also checked satellite images of two previous eruptions reported in news. Image acquired with Landsat 5 was influenced by sensor anomaly called \"oversaturation\". Sensor anomalies must be considered when interpreting satellite images. All known anomalies appearing in Landsat images are collected and described <a href=\"https://landsat.usgs.gov/known-issues\">here</a>. For Sentinel-2 satellite one can find similar anomalies described in <a href=\"https://sentinel.esa.int/web/sentinel/user-guides/sentinel-2-msi/document-library\">Data Quality Reports</a>.\n",
+    "We also checked satellite images of two previous eruptions reported in news. Image acquired with Landsat 5 was influenced by sensor anomaly called \"oversaturation\". Sensor anomalies must be considered when interpreting satellite images. All known anomalies appearing in Landsat images are collected and described <a href=\"https://www.usgs.gov/core-science-systems/nli/landsat/landsat-known-issues">here</a>. For Sentinel-2 satellite one can find similar anomalies described in <a href=\"https://sentinel.esa.int/web/sentinel/user-guides/sentinel-2-msi/document-library\">Data Quality Reports</a>.\n",
```

Comment:
I happened to glance upon this and the problem is that you are missing a `\` sign. Instead of `href=\"https://www.usgs.gov/core-science-systems/nli/landsat/landsat-known-issues"` you should have `href=\"https://www.usgs.gov/core-science-systems/nli/landsat/landsat-known-issues\"`.

That is because a jupyter notebook is essentially a JSON file and in JSON files characters `"` inside strings have to be written as `\"`.